inhouse_bot/cogs/queue_cog.py

```diff
@@ -529,7 +529,7 @@ async def verify(
 
         # Use icons 1 through 28 for verification -- excluding the current profile icon
```

Comment:
comment should also say 29 right?

Reply:
the range function is start inclusive, stop exclusive, so I needed to change it to range(1,29)